tests/init_db.sh

```diff
@@ -4,8 +4,8 @@
 datacube system init
 datacube system check
 
-datacube product add https://raw.githubusercontent.com/digitalearthafrica/config/master/products/esa_s2_l2a.odc-product.yaml
-stac-to-dc --bbox='4,5,5,6' --collections='sentinel-s2-l2a-cogs' --datetime='2019-09-01/2020-01-01' --limit 1000
+# datacube product add https://raw.githubusercontent.com/digitalearthafrica/config/master/products/esa_s2_l2a.odc-product.yaml
```

Comment:
FYI I think Damien fixed these failing datasets in [odc-tools by mocking them](https://github.com/opendatacube/odc-tools/commit/2b57d1594574b46c15203492dff72cb88cbce6d1#diff-55e5c5a9e1d9db731fbe83c8f3ea8d424f4d592e1a661b6c5e5008f78964af0cR63)

Reply:
one important point of using the real datasets here is to guarantee the sanity/match of both end, code and input, and alarmed by any violation. To mock the dataset and pass the basic test was done enough (or should have) in the unit test.